projects/hal-components/src/lib/components/table-viewer/table-engine/table-viewer.base.ts

```diff
@@ -32,6 +33,13 @@ export class TableViewerBase implements OnInit, AfterViewInit, OnChanges, OnDest
 
   initialTableDataLoaded = false;
 
+  slicedList: number[];
+  pagination: PaginationConfig = {
+    pageSize: 10,
+    pageIndex: 0,
+    length: 5
+  };
```

Comment:
Er det noe grunn til at den skal være definert her, blir jo definert i setupPagination()? Kan kanskje bare stå: pagination: PaginationConfig; Ser jeg har gjort det samme i paginator docen, så kanskje noe som kan endres der og. Men hvis det er noe grunn burde det gi mening? pageSize kan isåfall ikke være 10 siden length er 5